src/bundle/Resources/views/themes/admin/ui/component/dropdown.html.twig

```diff
@@ -60,7 +60,15 @@
                     />
                 {% endblock %}
             {%- endembed -%}
-            <ul class="ibexa-dropdown__items-list">
+            <ul
+                class="ibexa-dropdown__items-list"
+                data-item-template="{{ include('@ibexadesign/ui/component/dropdown_item.html.twig', {
```

Comment:
Here as well, it can just be data-template, after all it's template in `ibexa-dropdown__items-list` so I think it's obvious it's template for item in this list